libs/lib-postgres/src/db/connection/ConnectionSlot.ts

```diff
@@ -78,19 +76,6 @@ export class ConnectionSlot extends framework.DisposableObserver<ConnectionSlotL
     }
   }
 
-  registerListener(listener: Partial<ConnectionSlotListener>): () => void {
-    const dispose = super.registerListener(listener);
-    if (this.connection && this.hasNotificationListener()) {
-      this.configureConnectionNotifications(this.connection);
```

Comment:
This could cause an uncaught exception if a client subscribed to the active checkpoint stream after the underlaying Postgres connection was internally closed after a timeout. Always `listen`ing to the notifications on connect should solve that issue.